assets/home/sections/sensei-pro-ad.js

```diff
@@ -138,40 +27,125 @@ const SenseiProAd = () => {
-				excerpt={ senseiProExtension.excerpt }
-				description={ __(
-					'By upgrading to Sensei Pro, you get all the great features found in Sensei LMS plus:',
-					'sensei-lms'
-				) }
-				features={ [
-					__( 'WooCommerce integration', 'sensei-lms' ),
-					__( 'Schedule ‘drip’ content', 'sensei-lms' ),
-					__( 'Set expiration date of courses', 'sensei-lms' ),
-					__( 'Advanced quiz features', 'sensei-lms' ),
-					__(
-						'Flashcard, image hotspot, and tasklist blocks',
-						'sensei-lms'
-					),
-					__( 'Premium support', 'sensei-lms' ),
-				] }
-				image={ senseiProExtension.image_large }
-				badgeLabel={ __( 'new', 'sensei-lms' ) }
-				price={ sprintf(
-					// translators: placeholder is the price.
-					__( '%s USD / year (1 site)', 'sensei-lms' ),
-					senseiProExtension.price
-				) }
-				buttonLink="https://senseilms.com/sensei-pro/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=extensions_header"
-				buttonTitle={ __( 'Learn More', 'sensei-lms' ) }
-			/>
+			<article className="sensei-home__sensei-pro-ad">
+				<section className="sensei-home__sensei-pro-ad__column">
+					<div className="sensei-home__sensei-pro-ad__content">
+						<header className="sensei-home__sensei-pro-ad__header">
+							<h2 className="sensei-home__sensei-pro-ad__title">
+								{ __(
+									'Start selling with Sensei Pro',
+									'sensei-lms'
+								) }
+							</h2>
+						</header>
+
+						<div className="sensei-home__sensei-pro-ad__description">
+							<p>
+								{ __(
+									'All the features in one package made so that you can start selling your courses right away.',
+									'sensei-lms'
+								) }
+							</p>
+
+							<div className="sensei-home__sensei-pro-ad__price">
+								<h3 className="sensei-home__sensei-pro-ad__price__title">
+									{ sprintf(
+										// translators: placeholder is the price.
+										__( '%s USD', 'sensei-lms' ),
+										senseiProExtension.price.replace(
+											'.00',
+											''
+										)
+									) }
+								</h3>
+								<p>
+									{ __( 'per year, 1 site', 'sensei-lms' ) }
+								</p>
+							</div>
+
+							<ul>
+								<li>
+									{ __(
+										'WooCommerce integration',
+										'sensei-lms'
+									) }
+								</li>
+								<li>
+									{ __(
+										'Schedule ‘drip’ content',
+										'sensei-lms'
+									) }
+								</li>
+								<li>
+									{ __(
+										'Set expiration date of courses',
+										'sensei-lms'
+									) }
+								</li>
+								<li>{ __( 'Quiz timer', 'sensei-lms' ) }</li>
+								<li>
+									{ __(
+										'Flashcards, Image Hotspots, Checklists and Interactive Video',
+										'sensei-lms'
+									) }
+								</li>
+								<li>
+									{ __(
+										'1 year of updates & support',
+										'sensei-lms'
+									) }
+								</li>
+							</ul>
+
+							<a
+								href="https://href.li/?https://senseilms.com/checkout/?add-to-cart=7009"
```

Comment:
Are we using `href.li` elsewhere? Hopefully the `noreferrer` will be enough.

Reply:
Good catch! Fixed in [42ac598](https://github.com/Automattic/sensei/pull/5987/commits/42ac59813594f5ce01051fa0c00aac2d9dca99a9). Thanks!